loadtest/control/simplecontroller/config.go

```diff
@@ -0,0 +1,50 @@
+package simplecontroller
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+// Config holds the the rate and user actions definitions that will be runned by
```

Comment:
"that will be run .."

And also a period at the end of the sentence.

And also, it would be nice if we could document the fields too since they are exported.